frontend/test/metabase/scenarios/auth/signin.cy.spec.js

```diff
@@ -53,4 +55,21 @@ describe("scenarios > auth > signin", () => {
     // order table should load after login
     cy.contains("37.65");
   });
+
+  sizes.forEach(size => {
+    it(`should redirect from /auth/forgot_password back to /auth/login (viewport: ${size})`, () => {
+      if (Cypress._.isArray(size)) {
+        cy.viewport(size[0], size[1]);
+      } else {
+        cy.viewport(size);
+      }
+
+      cy.visit("/");
+      cy.url().should("contain", "auth/login");
+      cy.findByText("I seem to have forgotten my password").click();
+      cy.url().should("contain", "auth/forgot_password");
+      cy.findByText("Back to login").click();
```

Comment:
`"Back to login"` is covered by another element. Use `.click({ force: true });` to actually click the text.

Reply:
But isn't this going to render the whole test pointless?

Reply:
Ah, you're right. I was looking at the video replay, but on opening it in the test window I see what you mean.
It seemed from your description above as though you had solved the issue, but the test fails when I run it. Is that intentional?

Reply:
@Opalevanescence - as explained [here](https://github.com/metabase/metabase/pull/12894#issuecomment-657898235), the test **should** fail originally. If you have this branch locally, you should also have commit 7abad70774 (please confirm that you do?). After those changes were introduced, the test should pass. If you look closely, I've also included screenshots as comments to my original commits.

So, if tests are still failing for you, please share more details so that someone else may try to reproduce it.

Reply:
Yes, I see that. However, despite being on this branch, having those commits, and rebuilding my frontend, the test is failing. I also tried pausing the test and manually clicking it, but that' didn't work either. Maybe @flamber can have a look at this later?
Here's a screenshot to show that I have the commit.
<img width="580" alt="Screen Shot 2020-07-14 at 11 11 19 AM" src="https://user-images.githubusercontent.com/58329466/87461086-c2c15980-c5c2-11ea-8aee-0f83023e1e41.png">
 Below is a screenshot of the failing test.
<img width="1022" alt="Screen Shot 2020-07-14 at 11 08 26 AM" src="https://user-images.githubusercontent.com/58329466/87461018-a1606d80-c5c2-11ea-9bb5-a2c0e47f7ee1.png">

Reply:
Forgive me for possible ignorance (I am still learning about the system), but isn't the CircleCi also running all of the Cypress tests and it shows the green checkmark for this PR - meaning they have all passed?

Reply:
The CircleCI runs through many of the Cypress tests (this one included), but not all. In other words, this test is passing in CircleCI, but I would like to find out why it's failing locally before merging with `master`.

Reply:
@Opalevanescence Don't know if it is something to do with Cypress' integration with Chrome, but in the built-in Electron, it works. And testing it manually (not Cypress) in Chrome+Firefox also works as expected.
![image](https://user-images.githubusercontent.com/1447303/87479574-0a300000-c62c-11ea-9af6-cfff2f83d972.png)

Reply:
Okay; I'll merge the branch and dig into why mine isn't working tomorrow.